src/rantanplan/core.py

```diff
@@ -99,69 +99,96 @@ def have_prosodic_liaison(first_syllable, second_syllable):
             and second_syllable['syllable'][0] in LIAISON_SECOND_PART)
 
 
-def get_synalephas(words):
+def get_synalephas(sinaereris_words_dictionary):
+    """
+    Gets a list of dictionaries for each word on each line of the poem
+    and joins the syllables to create phonologic syllables according to its
```

Comment:
I've seen you've changed `phonologic` to `phonological` in some places. Let's try to keep some consistency with naming.